src/crypto.test.ts

```diff
@@ -161,11 +153,14 @@ describe("hashRegistryValue", () => {
 
     // "h" is the hash generated by siad with the same input parameters
     const h = "ff3b430675a0666e7461bc34aec9f66e21183d061f0b8232dd28ca90cc6ea5ca";
-    const hash = hashRegistryEntry({
-      datakey: "HelloWorld π",
-      data: "abc π",
-      revision: BigInt(123456789),
-    });
+    const hash = hashRegistryEntry(
```

Comment:
Maybe add a test case where `hashDataKeyHex` is `true`, asserting the output is what we expect?
Or assert that 
```
hashRegistryEntry(
      {
        datakey: "HelloWorld",
        data: "abc",
        revision: BigInt(123456789),
      },
      false
    )
``` 
!= 
```
hashRegistryEntry(
      {
        datakey: "HelloWorld",
        data: "abc",
        revision: BigInt(123456789),
      },
      true
    )
```